src/components/molecules/NavBar/NavBar.scss

```diff
@@ -165,7 +165,7 @@ $volume-popover-width-without-sliders: 15em;
   flex-direction: column;
   align-items: center;
   justify-content: center;
-  top: $navbar-height;
+  top: $navbar-height + 50px;
```

Comment:
If you add plain values, it is hard to figure out why exactly that `50px` and not `42px`. There are constants just like `$navbar-height` that will inform development of the reason for their use. This one can be as well.